### ISSUE TITLE
Redirection Error

### ISSUE THREAD

**Issue by enimajd93** [johngodley/redirection#632]
```
Plugin: 3.0
WordPress: 4.8.2
PHP: 5.6.33
Browser: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_5) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/63.0.3239.132 Safari/537.36

Error: Unexpected token < in JSON at position 0
Action: redirection/v1/redirect?orderby=id GET
Code: 404 Not Found
Raw: <!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
<html><head>
<title>404 Not Found</title>
</head><body>
<h1>Not Found</h1>
<p>The requested URL /pho_t3/index.php was not found on this server.</p>
<p>Additionally, a 404 Not Found
error was encountered while trying to use an ErrorDocument to handle the request.</p>
</body></html>


Error: Unexpected token < in JSON at position 0
Action: redirection/v1/group GET
Code: 404 Not Found
Raw: <!DOCTYPE HTML PUBLIC "-//IETF//DTD HTML 2.0//EN">
<html><head>
<title>404 Not Found</title>
</head><body>
<h1>Not Found</h1>
<p>The requested URL /pho_t3/index.php was not found on this server.</p>
<p>Additionally, a 404 Not Found
error was encountered while trying to use an ErrorDocument to handle the request.</p>
</body></html>

```

**Comment by johngodley** [comment]
What is `/pho_t3/index.php`? What is the URL of your site?

**Comment by johngodley** [comment]
You may find the 3.1 beta helps:

https://github.com/johngodley/redirection/releases/tag/3.1

**Comment by johngodley** [comment]
I've now released 3.1. I will close this ticket for now, but please reopen if the problem persists
